ushahidi/platform-client
# 2119 select language

This pull request makes the following changes:
- Settings no longer switches the language before saving your choice
- User doesn't need to log out for user language preference to persist
- user language preference saved to session

Testing checklist:
- add languages to .env file. for example: APP_LANGUAGES=en,es,ar
- Run a dist build to get languages
- Create new locales folder under app; Copy locales from servers/www/locales/ to app/locales/
- Run `gulp` normally
- [x] create new admin user
- [x] go to settings/general and then change language; hit save; language of site should switch to selected language;
- [x] log out; language of site should be in selected language
- [x] log in again; change language in mode-bar; language should change immediately. 
- [x] refresh page; language should persist
- [x] log out; language should switch to site default language
-------

Fixes ushahidi/platform#2119

Ping @ushahidi/platform
